src/main.js

```diff
@@ -198,6 +198,7 @@ function configureCommandlineSwitchesSync(cliArgs) {
 
 	// Read argv config
 	const argvConfig = readArgvConfigSync();
+	app.commandLine.appendSwitch('--autoplay-policy', 'no-user-gesture-required');
```

Comment:
Not really an issue with the code, but the location where you call this in the method seems strang to me. Above is about "Read argv config" and below it is being applied, but your change is not controlled by the argv.json file unless I am mistaken. 

You also seem to miss a comment as to why we are doing this, so a developer not familiar with this would have no clue as to why we do this.